doc/controller/controller.rst

```diff
@@ -1,3 +1,31 @@
-Controller class
+Controller
 =====================================
-To be filled.
+The controller is responsible for providing set points for the assets in the simulator.
+The controller parses the ESDL and creates objects of the assets it is providing set points for.
+When a time step is calculated the controller passes back a dict of the set points
+for the controllable assets in the network. The key of this dict is the id of the asset.
+The value is another dict. The key of this dict is the property which needs to be set
+(e.g. supply temperature, heat demand). The value is the set point for this property.
+The controller works based on the priority of the source. The controller will first allocate
+The controller will also  if the sources with priority 1 have enough capacity to provide the demand.
+if more capacity is required, the sources with priority 2 will be used etc. If the demand is lower
+then the available capacity of the observed sources, the remaining demand will be equally distributed
+over the source at the observed priority.
```

Comment:
So now we're implementing a very simple algorithm for the controller.   Does the current architecture allow for alternative control strategies?

Reply:
I added an abstract class which can serve as basis for other controllers